packages/notifier/test/test-notifier.js

```diff
@@ -1,6 +1,11 @@
 import { test } from './prepare-test-env-ava.js';
 import { makeNotifierKit } from '../src/index.js';
-import '../src/types-ambient.js';
+
+/**
+ * @import {ERef} from '@endo/far';
+ * @import {IterationObserver, LatestTopic, Notifier, NotifierRecord, PublicationRecord, Publisher, PublishKit, StoredPublishKit, StoredSubscription, StoredSubscriber, Subscriber, Subscription, UpdateRecord} from '../src/types.js';
```

Comment:
These long lists make me wonder about qualified imports.

Reply:
yeah. `.ts` and its tooling makes the long list easy to work with but for JSDoc we might be better off with a name per module